db_test.go

```diff
@@ -141,6 +142,17 @@ func newTestDB(t *testing.T, name string, applyOpts func(opts *Options)) *DB {
 	return db
 }
 
+func newSecondaryTestDB(t *testing.T, name string) *DB {
+	secondaryDir := name + "-secondary"
+
+	opts := NewDefaultOptions()
+	opts.SetMaxOpenFiles(-1)
```

Comment:
Is it needed? `-1` is the default https://github.com/facebook/rocksdb/blob/78afb4d81e757134b2dbf085cf0034ba7e1a3a44/include/rocksdb/options.h#L556

Reply:
Ah, so it is. I was following the wiki on how to utilize a secondary instance, where their example explicitly sets it to `-1`: https://github.com/facebook/rocksdb/wiki/Secondary-instance

Is it possible that for some reason the default could change in a future version? If so, it may make sense to leave this.